src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -253,6 +253,7 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
           Failure(regrets)
         case success => success
       }
+      case _ => //nothing
```

Comment:
To be honest I'm not sure why the case where a not-done workflow that doesn't have an id is not handled as well, because that should be aborted too.

Sounds good, I will add the comment